plugins/@grouparoo/spec-helper/src/lib/cliSpecHelper.ts

```diff
@@ -42,7 +42,11 @@ export namespace CLISpecHelper {
 
       const spawnProcess = spawn(command, args, {
         cwd,
-        env,
+        env: {
+          ...env,
+          GROUPAROO_TELEMETRY_DISABLED:
+            process.env.GROUPAROO_TELEMETRY_DISABLED,
+        },
       });
```

Comment:
Sometimes in tests we create totally fresh ENV (not relying on system env).  In these cases, we need to be sure to forward the previous value of `GROUPAROO_TELEMETRY_DISABLED`

cc @bleonard

Reply:
Why not just always "false" ? I think this context is always in test land